pkg/services/ngalert/store/alert_rule.go

```diff
@@ -42,7 +42,8 @@ type RuleStore interface {
 	GetAlertRules(ctx context.Context, query *ngmodels.GetAlertRulesQuery) error
 	GetUserVisibleNamespaces(context.Context, int64, *models.SignedInUser) (map[string]*models.Folder, error)
 	GetNamespaceByTitle(context.Context, string, int64, *models.SignedInUser, bool) (*models.Folder, error)
-	UpsertAlertRules(ctx context.Context, rule []UpsertRule) error
+	InsertAlertRules(ctx context.Context, rule []UpsertRule) error
```

Comment:
Can we change []UpsertRule to just []AlertRule? I can't see any reason why we should keep using it here because `Upsert.Update` is always nil.

Reply:
Good catch, will change it.